script/Arcueid/Day 13/13_00_ARC13_6.txt

```diff
@@ -47,7 +47,7 @@ How pathetically he clings to life.
 [sha:484bd2bc590af8caecd31796fd42a4cf074c9e02]{
 -- Page 449, Offset 32971.
 -- 倒れこむ一歩手前の体を引きずって、瓦礫を乗り越える。
-My body is on the verge of collapse, but I drag myself up and over the heaps of rubble.
+His body is on the verge of collapse, but he drags himself up and over the heaps of rubble.
```

Comment:
The 倒れこむ一歩手前 makes me think this is Shiki - that feels more like an internal assessment. Does the CG viewpoint change after this line? I read it as him going over the rubble to go see what happened to Roa, since he doesn't remark on his body til after

Reply:
<https://youtu.be/ow_e_g3Wy8U?t=6869> for the direct scene in question.

The main reason why I felt this line was about Roa was because of the very visceral sound of him dragging his body amongst rubble, with no Shiki movement sounds. Also, I find it makes more sense as a continuous description of Roa, rather than Shiki cutting in between to describe his own body about to collapse.

Reply:
Hmm, I dunno, the viewpoint change right after that line and the 一歩手前 still has me thinking it's him. @Avuxo, thoughts?

Reply:
If you interpret 倒れこむ as some sort of poetic way of describing _death_, the line makes more sense referring to Roa. However, I _do_ kinda doubt that interpretation from just a jp standpoint. 

From a _meaning_ standpoint, I can't argue with Yusuke's line of thought. In game it seems pretty clear that he's describing Roa. As such, I think the change should stay.

Reply:
Fair enough, I can see it. Merge away